src/tests/parser_test.py

```diff
@@ -416,6 +417,46 @@ def test_raise_from():
     parser.parse(code, 'file_path')
 
 
+@pytest.mark.skipif(six.PY2, reason='Matrix multiplication operator is '
+                                    'invalid in Python 2.x')
+def test_simple_matrix_multiplication():
+    """Make sure 'a @ b' doesn't trip the parser."""
+    if sys.version_info.minor < 5:
+        return
+    parser = Parser()
+    code = CodeSnippet("""
+        def foo():
+            a @ b
+    """)
+    parser.parse(code, 'file_path')
```

Comment:
The fact that `file_path` appears in every test is what bothers me, not the repetition of this call.